v2/internal/frontend/desktop/windows/win32/theme.go

```diff
@@ -50,6 +59,16 @@ func SetTheme(hwnd uintptr, useDarkMode bool) {
 	}
 }
 
+func EnableTranslucency(hwnd uintptr, backdrop BackdropType) {
+	fmt.Printf("%+v\n", windowsVersion)
+	if IsWindowsVersionAtLeast(10, 0, 22579) {
+		dwmSetWindowAttribute(hwnd, DwmwaSystemBackdropType, unsafe.Pointer(&backdrop), unsafe.Sizeof(&backdrop))
```

Comment:
Shouldn't we pass `unsafe.Sizeof(backdrop)` instead of the size of the pointer? Docs state it should correspond to the size of the value pointed to.

Reply:
Luckily I think they're the same. Will change though

Reply:
I have updated this. I was getting really weird behaviour but it turned out to be the size of the dark mode variable used before this one.